app/main/lib/presto.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+from flask import current_app as app
+import requests
+
+class Presto:
```

Comment:
... or is the idea that each service using alegre would have its own set of callback endpoints?

Reply:
I think for now, lets leave it as presto, and when we need to genericize it because we have two callbackable services, we genericize it then as a subclass setup.